packages/app/obojobo-repository/shared/components/notification.scss

```diff
@@ -0,0 +1,57 @@
+@import '~styles/includes';
+
+$poppin-pink: #af1b5c;
```

Comment:
Since this color is used in two places, remove the declarations from the individual stylesheets and put them in a single location used by everything that requires the color.

Also name more descriptively, e.g. `$color-notification`.